src/Card/README.md

```diff
@@ -613,7 +693,8 @@ it is meant to be used as a single horizontal row of Cards, not as a grid. See C
 ```jsx live
 <CardDeck>
   <Card>
-    <Card.ImageCap 
+    <Card.ImageCap
+      skeletonHeight={140}
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` prop here?

Reply:
Removed